### PR TITLE
Include cheats for pubg mobile

### PR DESCRIPTION
**Pull request by ariollex1**

### CODE REVIEW

**Review by ItsVixano — APPROVED** [review]
perfect , better that my pubg hack implementation "sniff"

**Review by RealAkito — APPROVED** [review]
Better than @Dyneteve original implementation, great job.

**Review by SebaUbuntu — CHANGES_REQUESTED** [review]
Good

File: atman_violet.mk [hunk 1]
```diff
@@ -8,17 +8,18 @@ $(call inherit-product, device/xiaomi/violet/device.mk)
 
 # Inherit some common Lineage stuff.
 $(call inherit-product, vendor/aosp/config/common.mk)
+$(call inherit-product, vendor/pubg/config/cheats.mk)
```
Comment (SebaUbuntu) [inline atman_violet.mk:11]: This repo is not cloned by default, you should add it to aosp.dependencies

**Review by SebaUbuntu — CHANGES_REQUESTED** [review]

**Comment by Keertesh19** [comment]
Pro 101 :3

File: atman_violet.mk [hunk 2]
```diff
@@ -8,17 +8,18 @@ $(call inherit-product, device/xiaomi/violet/device.mk)
...
-TARGET_BOOT_ANIMATION_RES := 1080
+TARGET_BOOT_ANIMATION_RES := 9999
 
 # Device identifier. This must come after all inclusions.
 PRODUCT_NAME := atman_violet
 PRODUCT_DEVICE := violet5G
-PRODUCT_BRAND := AtmanOS-5G
-PRODUCT_MODEL := ATMANOS NOTE 7 MEME 5G edition
+PRODUCT_BRAND := Xiaomeme
+PRODUCT_MODEL := PUBG MOBILE 7 Pro 5G Edition
```
Comment (RealAkito) [inline atman_violet.mk:22]: how dare u touch the model meme

Comment (ItsVixano) [reply atman_violet.mk:22]: this is important sar 
without that , we cannot hack **PABG**

**Pull request merged**